Changes.md

```diff
@@ -4,6 +4,10 @@ This file is a manually maintained list of changes for each release. Feel free
 to add your changes here when sending pull requests. Also send corrections if
 you spot any mistakes.
 
+## HEAD
+
+* Fix Amazon RDS profile for MySQL 5.6.x with 2019 CA #2292
```

Comment:
Perhaps a more apt comment would be "Fix Amazon RDS profile for engine versions that use yaSSL instead of OpenSSL with 2019 CA".

I confirmed that I could get older versions of 5.7 to fail too, and newer versions of 5.6 did work.

Reply:
Ah, you're right, my bad :) Let me update 👍 to try and keep it short, what about `Fix Amazon RDS profile for yaSSL MySQL servers with 2019 CA`?

Reply:
👍